### PR TITLE
Fix climateSettingSchedules.delete type

### PR DESCRIPTION
**Pull request by Andrewjeska**

### CODE REVIEW

File: src/types/route-responses.ts [hunk 1]
```diff
@@ -172,3 +172,5 @@ export interface ClimateSettingScheduleCreateResponse {
 export interface ClimateSettingScheduleUpdateResponse {
   climate_setting_schedule: ClimateSettingSchedule
 }
+
+export type ClimateSettingScheduleDeleteResponse = Record<string, never>
```
Comment (razor-x) [inline src/types/route-responses.ts:176]:
```suggestion
export type ClimateSettingScheduleDeleteResponse = null
```

File: src/seam-connect/routes.ts [hunk 2]
```diff
@@ -635,7 +636,7 @@ export abstract class Routes {
           }
         ),
       delete: (params: ClimateSettingScheduleDeleteRequest) =>
-        this.makeRequest({
+        this.makeRequest<ClimateSettingScheduleDeleteResponse>({
```
Comment (razor-x) [inline src/seam-connect/routes.ts:639]: We should probably do

```
        await this.makeRequest<ClimateSettingScheduleDeleteResponse>({
        ...
        return null
```

Comment (Andrewjeska) [reply src/seam-connect/routes.ts]: [7f41a70](https://github.com/seamapi/javascript/pull/263/commits/7f41a70b01e2a469831fbf0b4a5cb9b3fb2b76bb)

File: src/seam-connect/routes.ts [hunk 3]
```diff
@@ -635,12 +634,14 @@ export abstract class Routes {
             data: params,
           }
         ),
-      delete: (params: ClimateSettingScheduleDeleteRequest) =>
-        this.makeRequest<ClimateSettingScheduleDeleteResponse>({
+      delete: async (params: ClimateSettingScheduleDeleteRequest) => {
```
Comment (razor-x) [inline src/seam-connect/routes.ts:637]:
```suggestion
      delete: async (params: ClimateSettingScheduleDeleteRequest): Promise<ClimateSettingScheduleDeleteResponse> => {
```

**Review by razor-x — APPROVED** [review]

**Pull request merged**